src/frontend/App.tsx

```diff
@@ -55,13 +56,15 @@ const App = () => (
     <RunningStateProvider bridge={window.irsdkBridge}>
       <SessionProvider bridge={window.irsdkBridge} />
       <TelemetryProvider bridge={window.irsdkBridge} />
-      <HashRouter>
-        <EditMode>
-          <ThemeManager>
-            <AppRoutes />
-          </ThemeManager>
-        </EditMode>
-      </HashRouter>
+      <LapTimesStoreUpdater>
```

Comment:
I think we should only add this provider for the standings as that means we'll be tracking relative for every overlay.

The way it works currently unfortunately is because every overlay is a seperate browser window that means each overlay has independent set of providers given.

A better approach would just be to put it inside Standings.tsx or a container for it. It doesn't need to wrap its children or even be a component in this case just make it a hook.

Reply:
That makes sense, I'll make that update.

Reply:
I've made this change now, tested all fine.